sunbeam-python/sunbeam/plugins/interface/v1/openstack.py

```diff
@@ -124,16 +124,20 @@ def __init__(self, name: str, tf_plan_location: TerraformPlanLocation) -> None:
 
     def _get_tf_plan_full_path(self) -> Path:
         """Returns terraform plan absolute path."""
-        manifest_obj = Manifest.load_latest_from_clusterdb()
-        manifest_tfplans = manifest_obj.terraform
-        tfplan_dir = f"deploy-{self.tfplan}"
-        if manifest_tfplans and manifest_tfplans.get(tfplan_dir):
+        tfplan_dir = f"{self.tfplan}-plan"
+        try:
+            manifest_obj = Manifest.load_latest_from_clusterdb_on_default()
+            manifest_tfplans = manifest_obj.terraform
             return manifest_tfplans.get(tfplan_dir).source
-        elif self.tf_plan_location == TerraformPlanLocation.SUNBEAM_TERRAFORM_REPO:
-            return self.snap.paths.snap / "etc" / tfplan_dir
-        else:
-            plugin_class_dir = Path(inspect.getfile(self.__class__)).parent
-            return plugin_class_dir / "etc" / tfplan_dir
+        except Exception as e:
```

Comment:
Is it possible to have a more defined exception ?

For instance, if the connection to clusterdb failed, do we want to fallback to default plan, or abort ?

Reply:
I left without catching ClusterDB failures so that the command is aborted with exception.